### PR TITLE
Bug 1127762 - Fix intermittent failure in test_music_change_rating.py

### PR DESCRIPTION
**Pull request by viorelaioia-zz**

### CODE REVIEW

**Comment by viorelaioia** [comment]
http://jenkins1.qa.scl3.mozilla.com/job/flame-kk.ui.adhoc/650/console

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=230d8d2b7bf5)

**Comment by viorelaioia** [comment]
http://jenkins1.qa.scl3.mozilla.com/job/flame-kk.ui.adhoc/650/console

File: tests/python/gaia-ui-tests/gaiatest/apps/music/regions/player_view.py [hunk 1]
```diff
@@ -44,7 +44,9 @@ def tap_star(self, rate):
         """
 
         self.tap_cover_in_player_view()
-        self.marionette.find_element(*self._get_star_locator(rate)).tap()
+        star = self.marionette.find_element(*self._get_star_locator(rate))
+        Wait(self.marionette).until(expected.element_displayed(star))
```
Comment (bebef1987) [inline tests/python/gaia-ui-tests/gaiatest/apps/music/regions/player_view.py]: can we use something like ` Wait(self.marionette).until(expected.element_displayed(*self._get_star_locator(rate)))`

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=828c0a9b5327)

**Pull request merged**